src/NzbDrone.Core/Download/ProcessDownloadDecisions.cs

```diff
@@ -73,6 +73,7 @@ public ProcessedDecisions ProcessDecisions(List<DownloadDecision> decisions)
 
                 try
                 {
+                    _logger.Info("Downloading release {0} at priority {1}.", remoteEpisode.Release.Indexer, remoteEpisode.Release.IndexerPriority);
```

Comment:
Remove. DownloadReport already logs at Info.
Also {0} = Indexer name. so "Downloading release MyIndexer at priority x." whoopsie.
If you still want to log prio, log it at debug/trace level.

Reply:
Ok yeah that makes sense